test/envoye2e/stats_plugin/stats_test.go

```diff
@@ -261,7 +275,7 @@ func TestStatsGrpc(t *testing.T) {
 	params.Vars["ServerMetadata"] = params.LoadTestData("testdata/server_node_metadata.json.tmpl")
 	params.Vars["ServerHTTPFilters"] = params.LoadTestData("testdata/filters/stats_inbound.yaml.tmpl")
 	params.Vars["ClientHTTPFilters"] = params.LoadTestData("testdata/filters/stats_outbound.yaml.tmpl")
-
+	fmt.Println("THIS IS SERVERHTTPFILTERS", params.Vars["ClientHTTPFilters"])
```

Comment:
Needs some clean up. Looks like lots of place are not correctly formatted and debug log left

Reply:
okok on it